### PR TITLE
fixed cannot get application tts/stt speech credentials if there is n…

### PR DESCRIPTION
**Pull request by xquanluu**
…o verb's synthesizer/recognizer

Jambonz has 2 levels for synthesizer/recognizer: in application and inside a verb.

Currently, the label is following only verb's synthezier/recognizer, If verb does not provide any synthezier/recognizer, only vendor, language will use default application level but label does not use application's synethsizer/recognizer which is wrong.

### CODE REVIEW

File: lib/tasks/stt-task.js [hunk 1]
```diff
@@ -44,6 +44,9 @@ class SttTask extends Task {
       }
     } else {
       this.data.recognizer = {hints: [], altLanguages: []};
+      // If there is no verb's recognizer, use application's recognizer for label
```
Comment (davehorton) [inline lib/tasks/stt-task.js:47]: hmm, whoa.....default to a label named 'default' ?  Why is this necessary?  What if the user has a label actually named 'default'?

**Comment by vdharashive** [comment]
@xquanluu @davehorton if user set label then only its value need to be considered and it should not use any default from application level if its not set , last PR for label also failed some the test cases i am ok to revert that https://github.com/jambonz/jambonz-feature-server/commit/5ccea65b7f0e28ae58cf43dada40a5d6f53bec8e

File: lib/tasks/stt-task.js (hunk 1 above)
Comment (xquanluu) [reply lib/tasks/stt-task.js:47]: the logic later will check if label === default, then use cs.syntheizerLabel

We need to have a condition to identify when to use application default label.

**Comment by xquanluu** [comment]
by default, without any verb synthezier or recognizer, we have to use application.systhesizer/recognizer label

File: lib/tasks/stt-task.js (hunk 1 above)
Comment (xquanluu) [reply lib/tasks/stt-task.js:47]: By using this special string: {jambonz_label}

If there is no verb synthesizer/recognizer, jambonz will replace this special string to application synthesizer/recognizer label.

**Comment by davehorton** [comment]
@xquanluu can you rebase?  Also we should discuss the overall approach (the special case logic around a "magic label")

**Comment by vdharashive** [comment]
@xquanluu with google asr fallback is failing using config
